packages/console/src/containers/AppContent/components/Topbar/TenantSelector/index.tsx

```diff
@@ -0,0 +1,91 @@
+import { TenantTag } from '@logto/schemas/models';
+import classNames from 'classnames';
+import { useRef, useState } from 'react';
+import { useTranslation } from 'react-i18next';
+
+import TopbarDivider from '@/assets/images/divider.svg';
+import KeyboardArrowDown from '@/assets/images/keyboard-arrow-down.svg';
+import PlusSign from '@/assets/images/plus.svg';
+import Tick from '@/assets/images/tick.svg';
+import AppError from '@/components/AppError';
+import Divider from '@/components/Divider';
+import Dropdown, { DropdownItem } from '@/components/Dropdown';
+import useTenants from '@/hooks/use-tenants';
+import { onKeyDownHandler } from '@/utils/a11y';
+
+import TenantEnvTag from './components/TenantEnvTag';
+import * as styles from './index.module.scss';
+
+function TenantSelector() {
+  const { t } = useTranslation(undefined, { keyPrefix: 'admin_console' });
+  const { tenants, currentTenant: currentTenantInfo, currentTenantId, error } = useTenants();
+
+  const anchorRef = useRef<HTMLDivElement>(null);
+  const [showDropdown, setShowDropdown] = useState(false);
+
+  if (error) {
+    return <AppError errorMessage={error.message} callStack={error.stack} />;
+  }
+
+  if (!tenants?.length) {
+    return null;
+  }
+
+  return (
+    <>
+      <TopbarDivider />
+      <div
+        ref={anchorRef}
+        tabIndex={0}
+        className={styles.currentTenantCard}
+        role="button"
+        onKeyDown={onKeyDownHandler(() => {
+          setShowDropdown(true);
+        })}
+        onClick={() => {
+          setShowDropdown(true);
+        }}
+      >
+        <div className={styles.name}>{currentTenantInfo?.name ?? 'My project'}</div>
+        <TenantEnvTag
+          className={styles.tag}
+          tag={currentTenantInfo?.tag ?? TenantTag.Development}
+        />
+        <KeyboardArrowDown className={styles.arrowIcon} />
+      </div>
+      <Dropdown
+        hasOverflowContent
```

Comment:
This should not be needed, I think

Reply:
Seems its by design
<img width="317" alt="image" src="https://github.com/logto-io/logto/assets/15182327/76e08b05-9972-4cf9-92f9-7afe16b2d24c">